ros/src/tl_detector/tl_detector.py

```diff
@@ -190,7 +189,7 @@ def process_traffic_lights(self):
         light = None
         light_wp = -1
 
-        if self.pose:
+        if self.pose and not(self.waypoints is None):
```

Comment:
also u need to check self.pose is not None,
why not to use `self.waypoints is not None`?